packages/babel-plugin-transform-es2015-modules-umd/src/index.js

```diff
@@ -65,23 +74,74 @@ export default function ({ types: t }) {
             } else if (arg.value === "exports") {
               return t.memberExpression(t.identifier("mod"), t.identifier("exports"));
             } else {
-              let requireName = basename(arg.value, extname(arg.value));
-              let globalName = browserGlobals[requireName] || requireName;
+              let memberExpression;
 
-              return t.memberExpression(t.identifier("global"), t.identifier(
-                t.toIdentifier(globalName)
-              ));
+              if (state.opts.exactGlobals) {
+                let globalRef = browserGlobals[arg.value];
+                if (globalRef) {
+                  if (globalRef.indexOf(".") > -1) {
```

Comment:
It doesn't seem like this is necessary, since `.split(".")` on something with no `.` will do exactly what the `else` is doing anyway.